tests/FINDOLOGIC/Export/Tests/ExporterTest.php

```diff
@@ -26,4 +26,13 @@ public function testUnknownExporterTypeMustThrowException()
             $this->assertEquals('Unsupported exporter type.', $e->getMessage());
         }
     }
+
+    public function testCsvHeadingIsNotWrittenToOutputWhenStartIsqNotZero()
```

Comment:
* What's that stray `q` doing there?
* The heading line was already not being written if start was non-zero - as far as I can see, this change only refers to property column names. If I'm wrong, feel free to change the name of the PR back to the original.

Reply:
>  - What's that stray q doing there?

Spelling issue. Fixed!

> - The heading line was already not being written if start was non-zero - as far as I can see, this change only refers to property column names. If I'm wrong, feel free to change the name of the PR back to the original.

As the property column names are also part of the CSV-Heading I would say the name of the test is corresponding to what its being tested against.